workflow/spec/schema/serverless-workflow-schema-01.json

```
@@ -4,11 +4,29 @@
   "description": "Workflow is a vendor-neutral specification for defining the format/primitives that the users can use to specify/describe their serverless application flow.",
   "type": "object",
   "properties": {
+    "id": {
+      "type": "string",
+      "description": "Workflow unique identifier. Conforms the universally unique identifier (UUID) format.",
```

We should not enforce UUID format. As long as it has simple characters 
```suggestion
      "pattern": "$[a-zA-Z0-9\-\.]+^",
      "description": "Workflow identifier. Accepted characters include both uppercase and lowercase ASCII letters, the digits 0 through 9, space, dash and dot: A-Z, a-z, 0-9, -, .",
```